secure_message/application.py

```diff
@@ -7,7 +7,7 @@
 from flask import json, jsonify
 from flask_restful import Api
 from flask_cors import CORS
-import redis
+import maya
```

Comment:
Some other ras repos use iso8601 (if that's right) should they all be using the same datetime stuff, or is this for something else?

Reply:
This is something else. They probably shouldn't be using that anyway - it's not difficult to parse a datestring using stdlib (or indeed, Maya).